test/payment/eip712/ForwardRequest.eip712.js

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
recommend to move this file somewhere else than in payment folder as it is only related to meta-tx